Tests/test_content.py

```diff
@@ -411,14 +413,26 @@ def run_test_scenario(t, c, proxy, default_test_timeout, skipped_tests_conf, nig
             return
 
     # Skip bad test
-    if playbook_id in skipped_tests_conf.keys():
+    if playbook_id in skipped_tests_conf:
         skipped_tests.add("{0} - reason: {1}".format(playbook_id, skipped_tests_conf[playbook_id]))
         return
 
     # Skip integration
     if has_skipped_integration:
         return
 
+    # Skip version mismatch test
+    test_from_version = t.get('fromversion', '0.0.0')
+    test_to_version = t.get('toversion', '99.99.99')
+    if (server_version_compare(test_from_version, server_numeric_version) > 0
```

Comment:
mmm shouldn't it be `server_version_compare(test_from_version, server_numeric_version) <= 0`?

Reply:
no, negative means that the from version is lower that the server version. in this case we don't want to skip